src/Compute/Compute/help/New-AzVM.md

```diff
@@ -146,7 +146,7 @@ $VirtualMachine = Set-AzVMOperatingSystem -VM $VirtualMachine -Windows -Computer
 $VirtualMachine = Add-AzVMNetworkInterface -VM $VirtualMachine -Id $NIC.Id
 $VirtualMachine = Set-AzVMOSDisk -VM $VirtualMachine -Name $OSDiskName -VhdUri $OSDiskUri -SourceImageUri $SourceImageUri -Caching $OSDiskCaching -CreateOption $OSCreateOption -Windows
 
-New-AzVM -ResourceGroupName $ResourceGroupName -Location $LocationName -VM $VirtualMachine -Verbose
+New-AzVM -ResourceGroupName $ResourceGroupName -Location $LocationName -VM $VirtualMachine -Verbose -Zone @("1")
```

Comment:
>@("1") [](http://example.com/codeflow?start=106&length=6)

Do you really need this syntax? Normally PowerShell is pretty clever about:
1) Converting to from int to string
2) Making an array of 1 element out of a scalar value
3) Hardly, needs the @(...) notation: a comma separated list of values is more idiomatic

BTW, is cmdet really expecting a string (as opposed to an integer)?

IOW, unless there are reason against it, I would just go with a simpler:

```PowerShell
New-AzVM -ResourceGroupName $ResourceGroupName -Location $LocationName -VM $VirtualMachine -Verbose -Zone 1
```